### DISCUSSION TITLE
In version inav 7, when flashing speedybee f405 v4 firmware, the engines do not work.

### DISCUSSION THREAD

**Discussion by Voltenergy** [iNavFlight/inav-configurator#2167] [Q&A]
In the Outputs tab I can't turn on the engine. See screenshot. If the firmware is loaded correctly there should be 4 motors that can be turned on. 
![drivers](https://github.com/user-attachments/assets/21e57575-35a5-49f2-a8e5-0521b200c5d0)

**Comment by sensei-hacker** [comment]
In the mixer tab, load and apply a preset.

Or use Configurator 7.1.0

**Reply by Voltenergy** [reply]
Thank you, the problem was solved by reinstalling the configurator to version 7.1.0
